include/SugarFields/Fields/Html/EditView.tpl

```diff
@@ -0,0 +1,44 @@
+{*
+/**
+ *
+ * SugarCRM Community Edition is a customer relationship management program developed by
```

Comment:
IF this is a new file it do not require the full copyright notice as this is not a SugarCRM forked file but a new file.

Reply:
First time I hear this, not saying it isn't true but I've seen the notice included in every new file so far...

https://github.com/salesagility/SuiteCRM/pull/5669/files
https://github.com/salesagility/SuiteCRM/pull/5735/files

Just as examples... there is many more.

Reply:
I know its the actual way of doing things but it should be discussed.
Else what will be like in 10 years: plenty of Sugar notices on files not created with Sugar code! 
See a RFC https://github.com/salesagility/SuiteCRM/issues/5952

Reply:
I'm not entirely certain but it's my understanding that any modifications to core (even if it's a new file) would "inherit" the AGPL and thus require the license. Definitely something to investigate / discuss further though!

Reply:
Any file inherit the AGPL but not the "Appropriate Legal Notices" or Sugar references on all files after a fork.
For legal you can only say "licensed under AGPL" in the header and provide a full text copy as a file. Take a look into Joomla that was a fork from Mambo.
The GPL doesn't require advertisement, just disclosure
Anyway this is a Legal issue and should be taken in detail by SalesAgility.

Interesting light reading: https://opensource.stackexchange.com/questions/4971/what-are-the-attribution-requirements-of-the-agpl-license